tests/test_build_docs_version.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+import pytest
+
+from build_docs import Version
+
+
+def test_equality() -> None:
+    # Arrange
+    version1 = Version(name="3.13", status="stable", branch_or_tag="3.13")
+    version2 = Version(name="3.13", status="stable", branch_or_tag="3.13")
+
+    # Act / Assert
+    assert version1 == version2
+
+
+@pytest.mark.parametrize(
+    ("name", "expected"),
+    [
+        ("3.13", "-rrequirements.txt"),
+        ("3.10", "standard-imghdr"),
+        ("3.7", "sphinx==2.3.1"),
+        ("3.5", "sphinx==1.8.4"),
+    ],
+)
+def test_requirements(name: str, expected: str) -> None:
+    # Arrange
+    version = Version(name=name, status="stable", branch_or_tag="")
+
+    # Act
+    requirements = version.requirements
+
+    # Assert
+    assert expected in requirements
```

Comment:
```suggestion
    version = Version(name=name, status="stable", branch_or_tag="")
    requirements = version.requirements
    assert expected in requirements
```
This is probably just a matter of taste, but I'd rather being more mindful of vertical space and remove superfluous comments and whitespace.

Reply:
This whole test case fits on the screen, so I think we're fine with vertical space. 

The idea behind the arrange/act/assert pattern is that tests benefit from being extra explicit in what they're doing, so when they fail, you know if the failure is in the actual thing under test, or just setup code. I've worked on codebases with very long test cases and it wasn't clear at all what was what.

That said, I'll apply this:

```suggestion
    # Arrange
    version = Version(name=name, status="stable", branch_or_tag="")

    # Act / Assert
    assert expected in version.requirements
```